### PR TITLE
Improved the display names for the risk outputs

### PR DESCRIPTION
**Pull request by micheles**
Closes https://github.com/gem/oq-engine/issues/3032. You can see the outputs in https://travis-ci.org/gem/oq-engine/jobs/329844626.

### CODE REVIEW

**Comment by raoanirudh** [comment]
@micheles, hopefully the list below is exhaustive:

- [ ] agg_curves-rlzs
- [ ] agg_curves-stats
- [ ] agg_loss_table
- [ ] agglosses-rlzs
- [ ] agglosses-rlzs
- [ ] avg_losses-rlzs
- [ ] avg_losses-stats
- [ ] bcr-rlzs
- [ ] bcr-stats
- [ ] damages-rlzs
- [ ] damages-stats
- [ ] disagg
- [ ] dmg_by_asset
- [ ] fullreport
- [ ] gmf_data
- [ ] hcurves
- [ ] hmaps
- [ ] loss_curves-stats
- [ ] loss_maps-rlzs
- [ ] loss_maps-stats
- [ ] losses_by_asset
- [ ] losses_by_event
- [ ] realizations
- [ ] ruptures
- [ ] sourcegroups
- [ ] uhs

**Comment by micheles** [comment]
@raoanirudh there is also the manual to update.

File: openquake/server/db/actions.py [hunk 1]
```diff
@@ -237,7 +238,38 @@ def get_outputs(db, job_id):
...
+    #'damages_by_event': 'Aggregate Event Damages',
+    'losses_by_asset': 'Average Asset Losses',
+    'losses_by_event': 'Aggregate Event Losses',
+    'damages-rlzs': 'Asset Damage Distribution',
+    'avg_losses-rlzs': 'Average Asset Losses',
+    'avg_losses-stats': 'Average Asset Losses Statistics',
+    'loss_curves': 'Asset Loss Curves',
+    'loss_curves-stats': 'Asset Loss Curves Statistics',
+    'loss_maps-rlzs': 'Asset Loss Maps',
+    'loss_maps-rlzs': 'Asset Loss Maps Statistics',
```
Comment (raoanirudh) [inline openquake/server/db/actions.py]: Typo: Should be `loss_maps-stats`

File: openquake/server/db/actions.py [hunk 2]
```diff
@@ -237,7 +238,38 @@ def get_outputs(db, job_id):
...
 
-DISPLAY_NAME = dict(dmg_by_asset='dmg_by_asset')
+DISPLAY_NAME = {
+    'gmf_data': 'Ground Motion Fields',
+    'dmg_by_asset': 'Average Asset Damages',
+    #'damages_by_asset': 'Average Asset Damages',
+    #'damages_by_event': 'Aggregate Event Damages',
+    'losses_by_asset': 'Average Asset Losses',
+    'losses_by_event': 'Aggregate Event Losses',
+    'damages-rlzs': 'Asset Damage Distribution',
```
Comment (raoanirudh) [inline openquake/server/db/actions.py]: Still missing `damages-stats`

File: openquake/server/db/actions.py (hunk 1 above)
Comment (micheles) [reply openquake/server/db/actions.py]: Fixed in https://github.com/gem/oq-engine/commit/b9d79beae8d86fb95aea312730abc813d2fc31f6

**Pull request merged**